From b30fa07b737dd81b538db364c122772330a48dc5 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sat, 28 Oct 2023 12:26:15 +0100
Subject: [PATCH 1/1] prefs.js: Add a 0.75 option in animation scale

Impatience used 0.75 by default, provide that value on the scale to
help users move over.
---
 prefs.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/prefs.js b/prefs.js
index 73c525f..edc7725 100644
--- a/prefs.js
+++ b/prefs.js
@@ -49,7 +49,7 @@ export default class SpeedinatorPreferences extends ExtensionPreferences {
 		group.add(hbox);
 		page.add(group);
 
-		[0.25, 0.5, 1.0, 2.0].forEach(
+		[0.25, 0.5, 0.75, 1.0, 2.0].forEach(
 			mark => scale.add_mark(mark, Gtk.PositionType.TOP, "<small>" + mark + "</small>")
 		);
 		scale.set_value(settings.get_double("speed"));
@@ -57,4 +57,4 @@ export default class SpeedinatorPreferences extends ExtensionPreferences {
 			settings.set_double("speed", sw.get_value());
 		});
     }
-}
\ No newline at end of file
+}
-- 
2.42.0

